src/Workspaces/Core/Portable/Serialization/SerializableSourceText.cs

```diff
@@ -217,30 +217,31 @@ public TextLoader ToTextLoader(string? filePath)
     private sealed class SerializableSourceTextLoader : TextLoader
     {
         public readonly SerializableSourceText SerializableSourceText;
-        private readonly AsyncLazy<TextAndVersion> _lazyTextAndVersion;
+        private readonly VersionStamp _version = VersionStamp.Create();
```

Comment:
instead of the TextAndVersion lazy (but then held onto strongly in memory), we instead call through to the underlying SerializableSourceText to grab the TextAndVersion.  If not loaded, the SST will load it from hte backing MMF shared with the host process.  

Thsi allows the OOP process to dump texts when not needed, but to rehydrate them with no effort from the shared memory between the two.